service/http.proxy/src/test/java/org/kaazing/gateway/service/http/proxy/HttpProxyServiceTest.java

```diff
@@ -36,16 +36,16 @@ public void testCreateService() throws Exception {
         Assert.assertNotNull("Failed to create http.proxy service", service);
     }
 
-    @Test
-    public void shouldFailGatewayStartupAcceptAndConnectDifferentEndingSlashes() throws Exception {
+    @Test(expected = IllegalArgumentException.class)
```

Comment:
Not required and maybe not needed here, but consider using [junit expected exception rule](http://junit.org/junit4/javadoc/4.12/org/junit/rules/ExpectedException.html) to assert both the exception type and error message.  I think that would probably eliminate the need for the throw new AssertionError below